sdk/spring/azure-spring-boot-starter-storage/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
```

Comment:
Readme of storage starter is missing, and it's due to the wrong label of the storage starter inside the readme, please help to update it together.

Reply:
Updated.